pyscriptjs/src/remote_interpreter.ts

```diff
@@ -106,39 +117,22 @@ export class RemoteInterpreter extends Object {
         // TODO: Remove this once `runtimes` is removed!
         this.interpreter = this.interface;
 
+        this.FS.mkdirTree('/home/pyodide/pyscript');
+        this.FS.writeFile('pyscript/__init__.py', pyscript as string);
+        //Refresh the module cache so Python consistently finds pyscript module
+        this.invalidate_module_path_cache();
+
         this.globals = Synclink.proxy(this.interface.globals as PyProxyDict);
+        logger.info('importing pyscript');
+        this.pyscript_py = Synclink.proxy(this.interface.pyimport('pyscript')) as PyProxy & typeof this.pyscript_py;
```

Comment:
why do you have to use this type?
Shouldn't it be a `Synclink.Remote` or so?

Reply:
`Synclink.proxy` just puts a marker on the object, so the type changes to `orig_type & ProxyMarked`, i.e., `PyProxy & ProxyMarked`. But `PyProxy` allows indexing anything with return type `any`... which maybe is a bad decision. I gave `pyscript_py` a specific type `ProxyMarked & SomeMethods`. I cast it to the intersection of the current type and the type of `pyscript_py` and it gets implicitly upcast.

Reply:
See here:
https://github.com/pyodide/synclink/blob/main/src/async_task.ts#L485-L497